### PR TITLE
Parameterizing process-starter path and modify cpuset file path check

### PR DESCRIPTION
**Pull request by balintTobik**
- process-starter path can be set by new parameter in webhook
- cpuset file can also be found when container's id is not suffix of the folder's name

### CODE REVIEW

File: pkg/sethandler/controller.go [hunk 1]
```diff
@@ -244,8 +244,9 @@ func (setHandler *SetHandler) applyCpusetToContainer(containerID string, cpuset
 	trimmedCid := strings.TrimPrefix(containerID, "docker://")
 	var pathToContainerCpusetFile string
 	filepath.Walk(setHandler.cpusetRoot, func(path string, f os.FileInfo, err error) error {
-		if strings.HasSuffix(path, trimmedCid) {
+		if strings.Contains(path, trimmedCid) {
```
Comment (Levovar) [inline pkg/sethandler/controller.go]: why was this needed?

Comment (balintTobik) [reply pkg/sethandler/controller.go]: Because in current configuration cpuset files is under docker-<container_id>.scope folder. So directory name does not end with container_id.

File: cmd/webhook/webhook.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ import (
 var scheme = runtime.NewScheme()
 var codecs = serializer.NewCodecFactory(scheme)
 var resourceBaseName = "nokia.k8s.io"
+var ps_path string
```
Comment (TimoLindqvist) [inline cmd/webhook/webhook.go]: It's not mentioned in the contribution guide but camel case should be used in variable names. That seems to be current practice so let's follow it. I would also use more descriptive name here so it's clear what the purpose of the variable is.

Comment (balintTobik) [reply cmd/webhook/webhook.go]: Sorry! I forget that... But it is already corrected.

Comment (TimoLindqvist) [reply cmd/webhook/webhook.go]: Great !
